src/plans/index.ts

```diff
@@ -0,0 +1,106 @@
+import { ApiRequest, TVersion, Version } from '..';
+import {
+  Plan,
+  PlanCheckout,
+  PlanFeature,
+  PlanCapability,
+  PlanCurrency,
+} from '../types';
+import { v2PlanMethods } from './v2';
+
+export type PlanVersions = {
+  [Version.V2]: {
+    /**
+     *  Retrieves information about a plan by its UUID. By default, the response does not contain any relational data. If you want to expand the relational data, you can do so with the `expand` query parameter.
+     *
+     *  @param {string} planUuid - The UUID of the plan
+     *
+     * Docs - https://docs.salable.app/api/v2#tag/Plans/operation/getPlanByUuid
+     *
+     * @returns {Promise<IPlan>}
+     */
+    getOne: (
+      planUuid: string,
+      options?: {
+        expand?: string[];
+      },
+    ) => Promise<Plan>;
+
+    /**
+     * Retrieves a checkout link for a specific plan. The checkout link can be used by customers to purchase the plan.
+     *
+     * @param  {string} planUuid The UUID of the plan
+     *
+     * Docs - https://docs.salable.app/api/v2#tag/Plans/operation/getPlanCheckoutLink
+     *
+     * @returns {Promise<PlanCheckout>}
+     */
+    getCheckoutLink: (
+      planUuid: string,
+      options: {
+        successUrl: string;
+        cancelUrl: string;
+        granteeId: string;
+        member: string;
+        promoCode?: string;
+        allowPromoCode?: boolean;
+        customerEmail?: string;
+        customerId?: string;
+        currency?: string;
+        automaticTax?: string;
+        quantity?: string;
+        changeQuantity?: string;
+        requirePaymentMethod?: 'true' | 'false';
```

Comment:
This should be a boolean; any other parameters like this should also be a boolean.